hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/BlockStreamManagerImpl.java

```diff
@@ -203,6 +204,7 @@ public BlockStreamManagerImpl(
         this.blockPeriod = blockStreamConfig.blockPeriod();
         this.streamWriterType = blockStreamConfig.writerMode();
         this.hashCombineBatchSize = blockStreamConfig.hashCombineBatchSize();
+        this.streamBlockHeaderSeparately = blockStreamConfig.streamBlockHeaderSeparately();
```

Comment:
The reason we added a config here is to see the performance results?

Reply:
Yes, as this feature is somewhat of an optimization for the bandwidth, we want to be able to control it when evaluating the performance results.